Rainier-4U-MRW.xml

```diff
@@ -93820,6 +93388,10 @@
 		<id>IO_IOHS_PRE2</id>
 		<default>0</default>
 	</attribute>
+	<attribute>
+		<id>IO_IOHS_XTALK</id>
+		<default>NO_XTALK</default>
```

Comment:
I'm not expecting a new value for this to show up.  Please confirm the value is correct for the link that got changed.